atd-vzv/src/views/dashboard/fatalitiesByTimeOfDayWeek.js

```diff
@@ -0,0 +1,215 @@
+    .subtract(1, "month")
+    .format("MM");
+  const lastMonthLastDayNumber = moment(
+    `${thisYear}-${lastMonthNumber}`,
+    "YYYY-MM"
+  ).daysInMonth();
+  const lastMonthLastDayDate = `${thisYear}-${lastMonthNumber}-${lastMonthLastDayNumber}`;
+
+  const dayOfWeekArray = [
+    "Sunday",
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+    "Thursday",
+    "Friday",
+    "Saturday"
+  ];
+  const hourBlockArray = [
+    "12AM",
+    "01AM",
+    "02AM",
+    "03AM",
+    "04AM",
+    "05AM",
+    "06AM",
+    "07AM",
+    "08AM",
+    "09AM",
+    "10AM",
+    "11AM",
+    "12PM",
+    "01PM",
+    "02PM",
+    "03PM",
+    "04PM",
+    "05PM",
+    "06PM",
+    "07PM",
+    "08PM",
+    "09PM",
+    "10PM",
+    "11PM"
+  ];
+  let dataArray = [];
+
+  const getFatalitiesByYearsAgoUrl = () => {
+    if (dataView === 0) {
+      return `https://data.austintexas.gov/resource/y2wy-tgr5.json?$where=death_cnt > 0 AND crash_date between '${thisYear}-01-01T00:00:00' and '${lastMonthLastDayDate}T23:59:59'`;
+    } else {
+      let yearsAgoDate = moment()
+        .subtract(dataView, "year")
+        .format("YYYY");
+      return `https://data.austintexas.gov/resource/y2wy-tgr5.json?$where=death_cnt > 0 AND crash_date between '${yearsAgoDate}-01-01T00:00:00' and '${yearsAgoDate}-12-31T23:59:59'`;
+    }
+  };
+
+  const buildDataArray = () => {
+    dataArray = [];
+    hourBlockArray.forEach(hour => {
+      let hourObject = {
+        key: hour,
+        data: []
+      };
+      dayOfWeekArray.forEach(day => {
+        let dayObject = {
+          key: day,
+          data: 0
+        };
+        hourObject.data.push(dayObject);
+      });
+      hourObject.data.reverse();
+      dataArray.push(hourObject);
+    });
+  };
+
+  const calculatHourBlockTotals = data => {
+    buildDataArray();
+    data.data.forEach(record => {
+      const date = new Date(record.crash_date);
+      const dayOfWeek = date.getDay();
+      const time = record.crash_time;
+      const timeArray = time.split(":");
+      const hour = parseInt(timeArray[0]);
+      dataArray[hour].data[dayOfWeek].data++;
+    });
+    return dataArray;
+  };
+
+  useEffect(() => {
+    // Fetch records for selected year
+    axios.get(getFatalitiesByYearsAgoUrl(dataView)).then(res => {
+      setHeatmapData(calculatHourBlockTotals(res));
+    });
+  }, [dataView]);
+
+  const getYearsAgoLabel = yearsAgo => {
+    return moment()
+      .subtract(yearsAgo, "year")
+      .format("YYYY");
+  };
+
+  const [activeTab, setActiveTab] = useState("1");
+
+  const toggle = tab => {
+    if (activeTab !== tab) setActiveTab(tab);
+  };
+
+  return (
+    <Container>
+      <Row>
+        <h3>Traffic fatalities by time and day</h3>
+      </Row>
+      <Row>
+          <Heatmap
+            height={200}
+            width={400}
+            data={heatmapData}
+            series={<HeatmapSeries colorScheme={["#fee5d9", "#fcae91", "#fb6a4a", "#de2d26", "#a50f15"]} />}
```

Comment:
We should have our colors live in a separate js file as constants that can be imported anywhere into VZV. This will help us with consistency of our theme and color palette. 
ex: https://github.com/cityofaustin/atd-vz-data/blob/484_nonCR3_ui/atd-vzv/src/constants/colors.js

Reply:
Done - thanks!